plugins/web/opentelemetry-plugin-document-load/src/documentLoad.ts

```diff
@@ -184,7 +172,7 @@ export class DocumentLoad extends BasePlugin<unknown> {
       keys.forEach((key: string) => {
         if (hasKey(performanceNavigationTiming, key)) {
           const value = performanceNavigationTiming[key];
-          if (typeof value === 'number' && value > 0) {
+          if (typeof value === 'number' && value >= 0) {
```

Comment:
why `=` ? From what I remember when value was a 0 it simply meant like it doesn't exist / is nto available or was not measured. And also having the information that something lasted 0s might be not really useful. WDYT ?

Reply:
Specifically FireFox seems to have legit 0s in there with some frequency (for `fetchStart` possibly when loaded from cache? - not sure) and the `> 0` was preventing any doc load spans from being created.  I'm open to another rearranging of the logic to simply always report a doc load span as "best effort" based on known data, rather than keying the whole apparatus on whether or not a `fetchStart` was found.

Reply:
hmm fair point - it was rather to avoid of adding information that was irrelevant. I think can try to leave it then if it mean it will add some value.